modules/040-node-manager/hooks/update_node_group_status.go

```diff
@@ -356,6 +373,49 @@ func handleUpdateNGStatus(input *go_hook.HookInput) error {
 	return nil
 }
 
+func buildEvent(nodeGroup statusNodeGroup, msg string) *corev1.Event {
+	eventType := corev1.EventTypeWarning
+	reason := "MachineFailed"
+
+	if msg == "Started Machine creation process" {
+		eventType = corev1.EventTypeNormal
+		reason = "MachineCreating"
+	}
+
+	now := time.Now()
+
+	return &corev1.Event{
+		TypeMeta: v1.TypeMeta{
+			Kind:       "Event",
+			APIVersion: "v1",
+		},
+		ObjectMeta: v1.ObjectMeta{
+			Namespace:    "default",
```

Comment:
Events should be in `d8-cloud-instance-manager` namespace, where located `machine-controller-manager` `machine-deploloyments`, etc...

I think

Reply:
Oh, Event will be generated from NodeGroup, Ng is not namespaced object. I do not have any question.

But this event I would looking for in `d8-cloud-instance-manager` namespace :-)

Reply:
if you place this events into `d8-cloud-instance-manager` they will not be attached to a NodeGroup object. And when you make `kubectl describe ng worker` command, you will see no events